secure_redis/settings.py

```diff
@@ -4,12 +4,19 @@
 from django.core.exceptions import ImproperlyConfigured
 
 
-def get_secure_cache_opts():
+def get_secure_cache_opts(require_options=False):
```

Comment:
@GhalebKhaled Why do we use `require_options` flag? Is there any case where we need to call this method with the flag set to False?